src/test/scala/com/fulcrumgenomics/alignment/AlignerTest.scala

```diff
@@ -550,4 +551,37 @@ class AlignerTest extends UnitSpec {
 
     System.out.println(s"Run median=${counter.median()}, mean=${counter.mean()}")
   }
+
+  Seq(true, false).foreach { useDefault =>
+    "CachedAligner.align(Global)" should f"align two identical sequences with all matches (default constructor: $useDefault)" in {
```

Comment:
Is there any reason why we can't run all the unit tests over both aligner kinds?